src/syrupy/session.py

```diff
@@ -1,5 +1,4 @@
 import os
```

Comment:
This file is getting pretty large. Can we movem the data structures into a separate file / split up this file somehow?

Reply:
Done